config/resource-manager.hcl

```diff
@@ -221,7 +221,7 @@ service "netapp" {
 }
 service "networkfunction" {
   name      = "NetworkFunction"
-  available = ["2022-08-01"]
+  available = ["2022-11-01"]
```

Comment:
instead of changing this in place we should be adding the new version
```
available = ["2022-08-01", "2022-11-01"]
```

Reply:
Hi @katbyte, thank you for your reviewing. I have updated the configuration. Kindly help review it again. Thanks.